scripts/ldmx-env.sh

```diff
@@ -360,6 +372,40 @@ __ldmx_base() {
   return $?
 }
 
+###############################################################################                                                                    
+# __ldmx_setenv
+#   Tell us to pass an environment variable to the container when we run
+#   By default, we pass the LDMX_BASE and DISPLAY variables explicitly, 
+#   because their syntax is too different between docker and singularity,
+#   so none of these need to (or should be) specified.
+###############################################################################
+
+export LDMX_CONTAINER_ENVS=()
+__ldmx_setenv() {
+  local _env_to_set="$1"
+
+  if [[ $_env_to_set != *"="* ]]; then
+    echo "$_env_to_set doesn't follow the required syntax myEnv=someValue!"
+	echo "Not setting container environment variable $_env_to_set."
```

Comment:
Just fixing the tabs
```suggestion
	  echo "Not setting container environment variable $_env_to_set."
```